WebRoot/css/zm.css

```diff
@@ -1975,14 +1975,48 @@ div.CompDragArea * {
 	display:inline-block;
 }
 
-.newTaskBanner {
-	@InactiveCursor@
-	color:rgb(120, 111, 91);
-	padding:5px 0px;
+div.newTaskBanner div.ImgAdd-svg {
+	@TaskListViewAddNewTaskIcon@
+}
+
+.ZmTaskMultiView .DwtListView .DwtListView-ColHeader {
+	border-bottom:0px;
+}
+
+.ZmTaskMultiView .DwtListView  .RowDouble {
+	@TaskListViewItem@
```

Comment:
Not sure if due to this particular selector , but tried fixing things on the pr and identified that the preview pane bottom is somewhat broken with the changes.

<img width="1440" alt="screen shot 2017-06-09 at 12 50 44 pm" src="https://user-images.githubusercontent.com/4294012/26965182-47b4afc0-4d12-11e7-8c95-4267e28a8904.png">